contracts/ERC721AUpgradeable.sol

```diff
@@ -0,0 +1,508 @@
+// SPDX-License-Identifier: MIT
+// Creators: locationtba.eth, 2pmflow.eth
+// Made upgradeable by: mrmcgoats.eth for Xenum Technology (xenum.io)
```

Comment:
Let's keep the license and creator in-tact that follows the creator's guidelines. Appreciate your contribution, but not sure if we want that in the comments that are part of everyone's contracts moving forward. On the default `ERC721A.sol` file, it's simple like this:
```
// SPDX-License-Identifier: MIT
// Creator: Chiru Labs
```
Let's keep it that way